domains/cobralicious.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "cobralicious"
+    },
+    "records": {
+        "CNAME": "https://cobralicious.netlify.app/"
```

no https and trailing (/)
```suggestion
        "CNAME": "cobralicious.netlify.app"
```